scripts/Dockerfile

```diff
@@ -8,11 +8,12 @@ RUN make linux
 
 FROM alpine:${ALPINE_VERSION}
 RUN apk update && apk add --update ca-certificates libc6-compat
-COPY --from=build-env /go/src/github.com/Decentr-net/decentr/build/decentrd-linux-amd64 /usr/bin/decentrd
-COPY --from=build-env /go/src/github.com/Decentr-net/decentr/build/decentrcli-linux-amd64 /usr/bin/decentrcli
+COPY --from=build-env /go/src/github.com/Decentr-net/decentr/build/linux/decentrd /usr/bin/decentrd
 
 EXPOSE 26657
 EXPOSE 26656
+EXPOSE 9090
```

Comment:
what are these new ports for?

Reply:
According to config, but i see that for now 9090 and 9091 are redundant